guide/.vuepress/config.js

```diff
@@ -53,7 +66,7 @@ module.exports = {
 		repo: 'raspdevpy/ccdoc',
 		contributors:false,
 		search: true,
-		logo: 'https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp',
+		logo: '/favicon.ico',
```

Comment:
Did you check the resolution of this logo change?

Reply:
256x256px, is that okay?